code/lib/create-storybook/src/generators/baseGenerator.ts

```diff
@@ -169,8 +170,18 @@ const stripVersions = (addons: string[]) => addons.map((addon) => getPackageDeta
 const hasInteractiveStories = (rendererId: SupportedRenderers) =>
   ['react', 'angular', 'preact', 'svelte', 'vue3', 'html', 'solid', 'qwik'].includes(rendererId);
 
-const hasFrameworkTemplates = (framework?: SupportedFrameworks) =>
-  framework ? ['angular', 'nextjs'].includes(framework) : false;
+const hasFrameworkTemplates = (framework?: SupportedFrameworks) => {
+  if (!framework) {
+    return false;
+  }
+  // Nuxt has framework templates, but for sandboxes we create them from the Vue3 renderer
+  // As the Nuxt framework templates are not compatible with the stories we need for CI.
+  // See: https://github.com/storybookjs/storybook/pull/28607#issuecomment-2467903327
+  if (framework === 'nuxt') {
+    return process.env.IN_STORYBOOK_SANDBOX !== 'true';
+  }
```

Comment:
@tobiasdiez @hacknug I now only create the default stories (using those from the vue3 renderer) in the sandboxes.

Reply:
Sounds good to me!

Ideally, you would test against https://github.com/nuxt-modules/storybook/tree/main/examples/showcase, which has the highest coverage of features.